dinamic-posts/src/App.js

```diff
@@ -0,0 +1,96 @@
+import React, {Component} from 'react';
+// import './App.css';
+import PostList from './components/PostList';
+
+const serverUrl = 'https://jsonplaceholder.typicode.com/';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      posts: null,
+      users: null,
+      comments: null,
+      postList: null
+    }
+    this.loadData();
+  }
+
+  isLoading() {
```

Comment:
`isLoading ` - means boolean variable, not a function that return boolean

write as getter
```javascript
  get isLoading() {
```

or rename
```javascript
  getIsLoading() {
```